src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import App from 'components/App'
+import App from './components/App'
```

Comment:
Usá siempre rutas absolutas, así mantenés la consistencia con las demás

```suggestion
import App from 'components/App'
```